code/game/objects/effects/effect_system.dm

```diff
@@ -13,6 +15,19 @@ would spawn and follow the beaker, even if it is carried or thrown.
 	unacidable = 1//So effect are not targeted by alien acid.
 	pass_flags = PASSTABLE | PASSGRILLE
 
+
+/obj/effect/New()
+	.=..()
+	if (random_rotation)
+		var/matrix/M = transform
+		if (random_rotation == 1)
```

Comment:
Magic numbers!

Reply:
oops, should be TRUE, will fix

Reply:
Probably shouldn't. Exist case where `random_rotation == 2`